velox/dwio/common/tests/utils/E2EFilterTestBase.h

```diff
@@ -253,11 +261,12 @@ class E2EFilterTestBase : public testing::Test {
       int32_t rowIndex) {
     using T = typename TypeTraits<Kind>::NativeType;
     std::vector<vector_size_t> rows;
-    // The 5 first values are densely read.
+    // The first 5 values are densely read.
     for (int32_t i = 0; i < 5 && i < batch->size(); ++i) {
       rows.push_back(i);
     }
-    for (int32_t i = 5; i < 5 && i < batch->size(); i += 2) {
+    // The last 5 values are sparsely read.
+    for (int32_t i = 5; i < 15 && i < batch->size(); i += 2) {
```

Comment:
Remove debug code

Reply:
This is not debug code, this is to test whether the sparsely  reading of lines 5 to 15 meets expectations.